sdk/ai/Azure.AI.Projects/samples/Sample19_FineTuning_OSS.md

```diff
@@ -0,0 +1,165 @@
+# Open Source Model (OSS) Fine-Tuning
+
+This sample demonstrates how to create a fine-tuning job for open source models with GlobalStandard training type using the Azure AI Projects SDK.
+
+## Supported Models
+
+Supported models: Ministral-3B
+
+## Prerequisites
+
+- Install the Azure.AI.Projects package.
+
+- Set the following environment variables:
+  - `PROJECT_ENDPOINT`: The Azure AI Project endpoint, as found in the overview page of your Azure AI Foundry project.
+  - `MODEL_DEPLOYMENT_NAME`: The name of the model deployment to use for fine-tuning.
```

Comment:
- `TRAINING_FILE_PATH`: the file with training data.
- `VALIDATION_FILE_PATH`: the file with data for model validation.